README.md

```diff
@@ -514,6 +514,32 @@ $ mbed export -i uvision -m K64F
 
 Mbed CLI creates a `.uvprojx` file in the projectfiles/uvision folder. You can open the project file with uVision.
 
+### Serial terminal
+
+You can open a serial terminal to the COM port of a connected Mbed target (usually board) using the `mbed sterm` command. If no COM port is specified, Mbed CLI will attempt to detect the connected Mbed targets and their COM ports.
```

Comment:
> Mbed CLI will attempt to detect the connected Mbed targets and their COM ports.

Does this mean you can use the `mbed sterm` command with multiple Mbed boards plugged into the computer?

Reply:
Yes. It would detect all and iterate through them (one by one, not simultaneously).